src/components/Plants/usePlantPics.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const API_URL =
+  'https://www.flickr.com/services/rest/?method=flickr.photos.search&format=json&nojsoncallback=1';
+const API_KEY = process.env.NEXT_PUBLIC_FLICKR_API_KEY;
+
+// TODO add return type
+export default function usePlantPics(name: string) {
+  let [photos, setPhotos] = useState([]);
+
+  // TODO: use fetch instead?
+  useEffect(() => {
+    axios
```

Comment:
I tried to use `fetch()` first but encountered some issues. Can we try rewriting it someday?

Reply:
Sure, could you please describe issue?

Reply:
I will try to do this myself before our meeting, I've changed sooo much in the code since then that maybe the issue was somewhere else

Reply:
I rewrote it to fetch, no issues now.

Reply:
Maybe that was just an effect of coding after midnight 🤷🏻‍♀️

Reply:
I am sure it was a side effect ;)